core/opaevaluator.go

```diff
@@ -230,12 +237,19 @@ type Module struct {
 	Content string
 }
 
-func NewOPAModuleConfig(modules []Module) (*OPAModuleConfig, error) {
-	compiler := ast.NewCompiler().WithBuiltins(map[string]*ast.Builtin{
+func NewOPAModuleConfigWithBuiltins(modules []Module, builtinsDeclarations map[string]*ast.Builtin) (*OPAModuleConfig, error) {
+	builtins := map[string]*ast.Builtin{
 		custom_builtins.GetHeaderDecl.Name:     custom_builtins.GetHeaderDecl,
 		custom_builtins.MongoFindOneDecl.Name:  custom_builtins.MongoFindOneDecl,
 		custom_builtins.MongoFindManyDecl.Name: custom_builtins.MongoFindManyDecl,
-	})
+		audit.SetLabelsDecl.Name:               audit.SetLabelsDecl,
```

Comment:
Why added the audit here? It should not be passed by the sdk? Or it is usable by default?

Reply:
it is usable by default. I started working on a mechanism to provide custom builtins however core is not yet ready to support it, so I embedded the builtin function as a core-feature

Reply:
Ok, so it would be also in the future? 

The audit package is a public interface? Otherwise it could be put inside the internal folder

Reply:
specifically the "Audit" struct may be useful to be published.

I guess for now we can move it to internal and prevent it from being public in a way that we can break it

Reply:
moved to internal packages